compiler/src/dotty/tools/dotc/typer/Namer.scala

```diff
@@ -1392,7 +1392,7 @@ class Namer { typer: Typer =>
       case _ =>
         WildcardType
     }
-    paramFn(checkSimpleKinded(typedAheadType(mdef.tpt, tptProto)).tpe)
+    paramFn(checkSimpleKinded(typedAheadType(mdef.tpt, tptProto)).tpe.dropRepeatedAnnot)
```

Comment:
I would move this into `def widenRhs` above.

Reply:
Done.